Assets/Scripts/Lua/CLRBindings/LuaArenaStatus.cs

```diff
@@ -79,4 +79,19 @@ public class LuaArenaStatus {
         } else
             UnitaleUtil.DisplayLuaError("NOT THE WAVE SCRIPT", "sorry but pls don't");
     }
+
+    public void SetSprite(string filename) {
+        if (UIController.instance.GetState() == UIController.UIState.DEFENDING) ArenaManager.instance.sprite.Set(filename);
+        else                                                                    UnitaleUtil.DisplayLuaError("NOT THE WAVE SCRIPT", "sorry but pls don't");
```

Comment:
There's no reason for these to be unavailable outside of the wave script; the only reason why you can't move or resize the arena outside of waves is because of the engine handling it. Additionally, since you're already modifying the arena, please change the horrible errors if possible lol